### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by sirLisko**
**What**  
- added tab navigation to the search
- moved search as index
- enforced use of hackney font
- refactored header
- improved form in progress
- made header dynamic

### Details:
**Header**

Screenshot 2020-12-02 at 11 39 57
Screenshot 2020-12-02 at 11 40 05

**Tab navigation**
Screenshot 2020-12-02 at 11 40 12

**Incomplete forms**
Screenshot 2020-12-02 at 11 40 17

### CODE REVIEW

File: components/Layout/Header/Header.jsx [hunk 1]
```diff
@@ -1,57 +1,96 @@
...
-      </a>
-    </div>
-  );
+  const { asPath } = useRouter();
+  const [navLinks, setNavLinks] = useState(loggedNavLinks);
+  useEffect(() => {
+    if (!user) {
+      setNavLinks();
+    } else if (!getDataIncludes('/form')) {
+      setNavLinks(navLinks.filter(({ name }) => name !== 'Forms in progress'));
```
Comment (naomigreen) [inline components/Layout/Header/Header.jsx:38]: Getting an undefined error when navigating from the confirmation page to the home page

**Review by naomigreen — COMMENTED** [review]
Looks great. Navigating between forms and in-process needs updating as we're creating the form search page next

File: components/Layout/Header/Header.jsx (hunk 1 above)
Comment (sirLisko) [reply components/Layout/Header/Header.jsx:38]: Where exactly? I can't reproduce it 🤔

Comment (naomigreen) [reply components/Layout/Header/Header.jsx:38]: Fix working

**Review by naomigreen — APPROVED** [review]

**Pull request merged**